portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -29,4 +30,12 @@ public long getId() {
   public String getTimeStamp() {
     return this.timeStamp;
   }
+
+  public void setMaxComment(int maxComment) {
```

Comment:
What uses these? It seems weird that a comment object has a method that returns the maximum number of comments...

Reply:
Nothing uses that it was an idea to set a maxComment value to a comment that could later be used in the commentScript to display comments. However, it turns out that the problem is non existent in the Dev App Server and only exist on the public server so I will talk to support about it and if it cant be fix I plan to use this implementation